### PR TITLE
Minor docs update.

### PR DESCRIPTION
**Pull request by benjamin-tlon**

### CODE REVIEW

File: vere/pier.c [hunk 1]
```diff
@@ -408,7 +408,13 @@ _pier_work_send(u3_writ* wit_u)
...
 }
 
-/* u3_pier_work_save(): tell worker to save checkpoint.
+/*
+  u3_pier_work_save(): tell worker to save checkpoint.
+
+  If there are no unsnapshotted events, then do nothing, otherwise, send
+  `[%save ~]` to the slave.
+
+  XX Should we report success before we update `pir_u->sav_u->ent_d`?
```
Comment (joemfb) [inline vere/pier.c]: Upon further consideration, this comment would be clearer if it was something like: "Should we wait for some report of success before updating ..."

File: noun/events.c [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 #include "all.h"
 
-#if 0
+#ifdef U3_SNAPSHOT_VALIDATION
```
Comment (joemfb) [inline noun/events.c]: Should we add a build option and `#mesondefine` for `U3_SNAPSHOT_VALIDATION`?

File: noun/events.c [hunk 3]
```diff
@@ -746,51 +750,57 @@ _ce_image_fine(u3e_image* img_u,
...
+  - Verify the patch - because why not?
+  - Write the patch data into the image file. (This is idempotent).
+  - Delete and free the patches.
+
+  XX What does this mean? "This has to block the main thread.
+  All further processing can happen in a separate thread, though we
+  can't save again till this completes."
+
+  XX: When does `_ce_patch_compose` return NULL? When there's no loom
+  changes?
```
Comment (joemfb) [inline noun/events.c]: Right, when there are no dirty pages.

File: noun/events.c [hunk 4]
```diff
@@ -746,51 +750,57 @@ _ce_image_fine(u3e_image* img_u,
...
+  - Delete the patchfile and free it.
+  - Write all dirty pages to disk; clear protection and dirty bits.
+  - Sync the patch files to disk
+  - Verify the patch - because why not?
+  - Write the patch data into the image file. (This is idempotent).
+  - Delete and free the patches.
+
+  XX What does this mean? "This has to block the main thread.
+  All further processing can happen in a separate thread, though we
+  can't save again till this completes."
```
Comment (joemfb) [inline noun/events.c]: Once we've written the dirty pages to disk (and reset their dirty bits and protection flags), we could handle the rest of the checkpoint creation/processing in a new thread, or somehow in parallel with ongoing operations. As long as we don't try to save a new checkpoint until the in-progress checkpoint is complete.

File: noun/events.c (hunk 2 above)
Comment (benjamin-tlon) [reply noun/events.c]: Keeping around all these `#if 0` blocks is definitely an anti-pattern, but I'm not sure what the right thing is here. Is this something that we want to be able to turn on sometimes, or is it just deadish code?

Comment (joemfb) [reply noun/events.c]: I'm not sure. There's a lot of this sort of verification and debugging code around. It's probably enough to gate it with a specific `#define`, we can always add options later if we end up using this stuff. And if we never end up using it, we'll eventually just want to remove it.

**Pull request merged**